bvm/ballerina-profiler/src/main/java/io/ballerina/runtime/profiler/Main.java

```diff
@@ -0,0 +1,310 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.runtime.profiler;
+
+import io.ballerina.runtime.profiler.codegen.CustomClassLoader;
+import io.ballerina.runtime.profiler.codegen.MethodWrapper;
+import io.ballerina.runtime.profiler.util.CustomException;
+import org.apache.commons.io.FileUtils;
+
+import java.io.*;
```

Comment:
We need to avoid this kind of wildcard imports. You can also configure Intellij to avoid doing this and to only import the specific ones. Shall we change this?